### PR TITLE
special cases to parse borough from record_id

### PR DESCRIPTION
**Pull request by fvankrieken**
I moved the call to `get_boro` upstream to simplify the kpdb creation query a little. It checks whether borough is null, and then based on source parses borough from the record_id.

We could theoretically do this "first" (parse borough from record id, if null then call `get_boro`. But as you know I don't like that function to begin with, so I think this is fine for now

### CODE REVIEW

**Comment by fvankrieken** [comment]
After this, 9 records still need boro

| source | record_id | record_name |
|-|-|-|
| HPD Projected Closings | 3d3a2dd3c6ec864f5e18cec8e03eb2a7 | 784 COURTLANDT AVENUE |  
| HPD Projected Closings | ff4c3b3e09d4c3a0d527a5c7988242d0 | 1559 PROSPECT PLACE |  
| HPD RFPs | 0ce4a0321f576ad9b397dec9358f255f | MWBE Site B - 1921 Atlantic Ave |  
| HPD RFPs | 100c52f92560bc161659f99b08340e9f | 97 West 169 St |  
| HPD RFPs | 1db4d18c803723face138966e70095d8 | NYCHA West Brighton RFP |  
| HPD RFPs | 473c582483789bb07b5837ecdb746ae4 | Bedford-Stuyvesant Community Wealth and Wellness RFP (Fulton-Saratoga) |  
| HPD RFPs | 862c3d3640912f5ec6561375a03449ef | Hunters Point F + G |  
| HPD RFPs | 9bd199e88bb8dd9b128d0d3c71b520b5 | Hunters Point F + G |  
| HPD RFPs | defae821da6ac0b1a7610be5c7d5f484 | SustaiNYC (E. 111th Street) |

File: sql/create_kpdb.sql [hunk 1]
```diff
@@ -2,7 +2,12 @@ DROP TABLE IF EXISTS _kpdb;
 SELECT
     a.*,
     b.project_id,
-    get_boro(a.geom) as borough,
+    CASE 
+        WHEN a.borough IS NOT NULL THEN a.borough
```
Comment (mbh329) [inline sql/create_kpdb.sql:6]: This is great - honestly not quite familiar with casting the values to a json object so that's cool to see

Comment (mbh329) [reply sql/create_kpdb.sql:25]: running locally now

File: sql/join_boroughs.sql [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    a.*,
+    b.borocode::VARCHAR AS borough
+INTO _combined
+FROM combined a 
+LEFT JOIN dcp_boroboundaries b ON ST_intersects(a.geom, b.wkb_geometry);
+
+DROP TABLE combined;
+SELECT * INTO combined FROM _combined;
+DROP TABLE _combined;
+VACUUM ANALYZE combined;
```
Comment (fvankrieken) [inline sql/join_boroughs.sql:11]: I don't love this select into a new table and dump right back, but given that we're joining and getting new rows in the join it seemed like the simplest way to go. Open to suggestions here

**Comment by AmandaDoyle** [comment]
> After this, 9 records still need boro
| source | record_id | record_name | borough |
|-|-|-|-|
| HPD Projected Closings | 3d3a2dd3c6ec864f5e18cec8e03eb2a7 | 784 COURTLANDT AVENUE |  The Bronx |
| HPD Projected Closings | ff4c3b3e09d4c3a0d527a5c7988242d0 | 1559 PROSPECT PLACE |  Brooklyn |
| HPD RFPs | 0ce4a0321f576ad9b397dec9358f255f | MWBE Site B - 1921 Atlantic Ave |  Brooklyn |
| HPD RFPs | 100c52f92560bc161659f99b08340e9f | 97 West 169 St |  The Bronx |
| HPD RFPs | 1db4d18c803723face138966e70095d8 | NYCHA West Brighton RFP |  Brooklyn |
| HPD RFPs | 473c582483789bb07b5837ecdb746ae4 | Bedford-Stuyvesant Community Wealth and Wellness RFP (Fulton-Saratoga) |  Brooklyn |
| HPD RFPs | 862c3d3640912f5ec6561375a03449ef | Hunters Point F + G |  Queens |
| HPD RFPs | 9bd199e88bb8dd9b128d0d3c71b520b5 | Hunters Point F + G |  Queens |
| HPD RFPs | defae821da6ac0b1a7610be5c7d5f484 | SustaiNYC (E. 111th Street) |  Manhattan |

@levysamu  Can you confirm that the boroughs that I assigned to each project above is correct?  The borough value will need to be assigned to these projects via a manual correction.

**Comment by fvankrieken** [comment]
Part of #399

**Pull request merged**